### PR TITLE
Update to antifam Pfam scripts

### PR DESCRIPTION
**Pull request by tgrego**

### CODE REVIEW

File: PfamScripts/pud/pud-removeAntiFamMatches_new.pl [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+## $antifam_dir
+
+my $uniprot_seq_file = $pfamseq_dir . '/uniprot';
+
+if (! -e $uniprot_seq_file) {
+  $logger->logdie("Can't find seq file '$uniprot_seq_file': $!\n");
+}
+
+
+# chdir($work_dir)
```
Comment (jainamistry) [inline PfamScripts/pud/pud-removeAntiFamMatches_new.pl]: Remove these commented lines if not needed

File: PfamScripts/pud/pud-removeAntiFamMatches_new.pl [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+
+  foreach my $antifam_id (@{$antifam_list}) {
+    my $matches_file = $matches_dir . '/' . $antifam_id . '_matches';
+    my $log_file = $logs_dir . '/' . $antifam_id . '_log';
+    my $hmm_file = "${work_dir}/Antifam_hmms/${antifam_id}.hmm";
+
+    if (! -e $log_file) {
+      if ( ! grep /$antifam_id/, @running_jobs ) {
+        my $queue = $config->{farm}->{lsf}->{queue};
+        system("bsub -q $queue -R \"select[mem>2000] rusage[mem=2000]\" -n 8 -M 8000 -o $log_file -J ${antifam_id}_rAFM hmmsearch --cpu 8 --noali --cut_ga --tblout $matches_file $hmm_file $uniprot_seq_file")
```
Comment (jainamistry) [inline PfamScripts/pud/pud-removeAntiFamMatches_new.pl]: Might need to add a group (-g <group_name>) to the bsub command to stop too many jobs running at the same time. ~100 searches against a single fasta file is usually okay to run in parrallel. You can use the group /Pfam_100 (this groups allows a 100 jobs to be run concurrently).

File: PfamScripts/pud/pud-removeAntiFamMatches_new.pl [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+
+
+
+=head1 NAME
+
+pud-removeAntiFamMatches.pl
+
+=head1 SYNOPSIS
+
+  pirsf.pl -pfamseq_dir DIR [-work_dir DIR]
```
Comment (jainamistry) [inline PfamScripts/pud/pud-removeAntiFamMatches_new.pl]: It says pirsf.pl. Think you might have changed this in a later version.

File: PfamScripts/pud/pud-removeAntiFamMatches_new.pl [hunk 4]
```diff
@@ -41,11 +43,19 @@
 my $pfamDB = Bio::Pfam::PfamLiveDBManager->new( %{ $config->pfamliveAdmin } );
 my $dbh = $pfamDB->getSchema->storage->dbh;
 
+
+# testing, stop if not test db
```
Comment (jainamistry) [inline PfamScripts/pud/pud-removeAntiFamMatches_new.pl]: Remove this section

File: PfamScripts/pud/pud-make_uniprot_fasta.pl [hunk 5]
```diff
@@ -21,6 +20,11 @@
 my $pfamDB = Bio::Pfam::PfamLiveDBManager->new( %{ $config->pfamliveAdmin } );
 my $dbh = $pfamDB->getSchema->storage->dbh;
 
+
+# testing, stop if not test db
```
Comment (jainamistry) [inline PfamScripts/pud/pud-make_uniprot_fasta.pl:24]: Remove this section

File: PfamScripts/pud/pud-removeAntiFamMatches_new.pl [hunk 6]
```diff
@@ -369,26 +371,20 @@ sub copy_antifam_data {
 }
 
 
+sub help{
+  print STDERR << "EOF";
 
+This script creates a fasta file from the sequences in the uniprot
```
Comment (jainamistry) [inline PfamScripts/pud/pud-removeAntiFamMatches_new.pl]: I think this section has been copied from another script? It doesn't correspond to what is in the script

File: PfamScripts/pud/pud-removeAntiFamMatches_new.pl (hunk 2 above)
Comment (tgrego) [reply PfamScripts/pud/pud-removeAntiFamMatches_new.pl]: added the group

**Pull request merged**